scripts/onebranch/post-build.ps1

```diff
@@ -29,20 +29,59 @@ function CopySignedBinaries {
     Get-ChildItem -Path $BinDir -Recurse | Remove-Item -Force -Recurse
 }
 
+# List of binaries to copy to output directory
+$BinariesToCopy = @(
+    "ebpfapi.dll",
+    "ebpfapi.pdb",
+    "ebpfapi.lib",
+    "bpftool.exe",
+    "bpftool.pdb",
+    "ebpfnetsh.dll",
+    "ebpfnetsh.pdb",
+    "eBPFCore.sys",
+    "eBPFCore.pdb",
+    "ebpfsvc.exe",
+    "ebpfsvc.pdb"
```

Comment:
do we also need to include `netebpfext.sys`?

Reply:
Probably, let me fix that.